drv/cpu-seq-api/src/lib.rs

```diff
@@ -73,6 +73,28 @@ pub enum StateChangeReason {
     Overheat,
 }
 
+/// Indicates the result of a power state transition.
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    FromPrimitive,
+    Eq,
+    PartialEq,
+    IntoBytes,
+    Immutable,
+    KnownLayout,
+    Count,
+)]
+#[repr(u8)]
+pub enum Transition {
+    /// The sequencer successfully transitioned to the desired state.
+    Done = 1,
```

Comment:
Nit: if we're deliberately skipping value 0, there should be a comment explaining why.

(I'm also not sure if it's necessary, since this isn't deriving `IdolError`)

Reply:
Yeah, I don't actually think the representation of this matters, since it's never niche-optimized...I can just remove the `= 1`.

Reply:
removed it: 6454bb2697e6da700a7165715d64a505a4dfd345